src/handlers/shared/check-assignments.ts

```diff
@@ -0,0 +1,65 @@
+import { Context, IssueEvent } from "../../types";
+
+export async function checkPreviousAssignments(context: Context, sender: Context["payload"]["sender"]): Promise<boolean> {
+  const events = await getAssignmentEvents(context);
+  const senderLogin = sender.login.toLowerCase();
```

Comment:
I'm not sure if `sender` is always the user who created the comment, I think it's better to use `payload.comment.user.login` - this should be fixed across the whole codebase

Reply:
between the two `payload.comment.user.login` is the only one typed as the user potentially being `undefined` so I've used both opting for `comment.user` first